source/neuropod/internal/backend_registration.cc

```diff
@@ -186,10 +199,9 @@ bool register_backend(const std::string &    name,
     // Using OPE overcomes this problem
     if (registered_backends_by_type->find(type) != registered_backends_by_type->end())
     {
-        NEUROPOD_ERROR(
-            "Attempted to register a backend for type '{}', but one was already loaded. If you are trying "
-            "to use multiple versions of the same framework, you must use OPE. See the docs at https://neuropod.ai",
-            type);
+        NEUROPOD_ERROR("Attempted to register a backend for type '{}', but one was already loaded. If you are trying "
+                       "to use multiple versions of the same framework, you must use OPE. See the docs at https://neuropod.ai",
+                       type);
     }
```

Comment:
Not sure this formatting change is correct. Did you intend to add this to your commit?

Also, lint is still failing on this PR. Run `tools/autofix.sh` (and then `tools/lint.sh` to verify)